test/gui/tst_vfs/test.feature

```diff
@@ -8,8 +8,21 @@ Feature: Enable/disable virtual file support
 
     Scenario: Disable/Enable VFS
         Given user "Alice" has been created on the server with default attributes and without skeleton files
+        And user "Alice" has created folder "folder1" in the server
+        And user "Alice" has uploaded file with content "ownCloud" to "folder1/lorem.txt" in the server
+        And user "Alice" has uploaded file with content "Lorem Ipsum is simply dummy text of the printing and typesetting industry." to "testFile.txt" in the server
         And user "Alice" has set up a client with default settings
+        Then the file "testFile.txt" should exist on the file system
+        And the file "folder1/lorem.txt" should exist on the file system
         When the user disables virtual file support
         Then the "Enable virtual file support..." button should be available
+        And the file "folder1/lorem.txt" should exist on the file system with the following content
```

Comment:
is it possible to add the bytes assertion before checking content? IMO, that way we are asserting that the file is downloaded (with vfs disabled)